cmd/openshift-install/create.go

```diff
@@ -299,7 +299,7 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 			logrus.Fatal(err)
 		}
 		switch cmd.Name() {
-		case "cluster", "image":
+		case "cluster", "image", "pxe-files":
```

Comment:
For some reason, it says `INFO PXE-files created in: auth`. As an alternative, https://github.com/openshift/installer/pull/6699/files#diff-38f93f2da042069046ec25c748458e4aeb5457951a609feec974083616298f42R121

Reply:
Ah, right. It prints a list of directories (so we want it to say `auth, pxe`) **but** we use `PersistToFile()` instead of the regular mechanism for writing files, so maybe it doesn't get added to the list.